client/app/scripts/superdesk-archive/directives.js

```diff
@@ -243,6 +243,15 @@
                             familyService.fetchDesks(scope.item, false)
                                 .then(function(desks) {
                                     scope.desks = desks;
+                                    scope.tooltipText = '';
+
+                                    for (var i = 1; i < desks.length; i++) {
```

Comment:
Is `i`Initialized to 1 on purpose or should it be zero?
(BTW, using the built-in [Array.prototype.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) avoids these kinds of problems altogether, less error prone)